### PR TITLE
FE: manage booking

### PR DESCRIPTION
**Pull request by shawnohn**
image

### CODE REVIEW

File: client/src/pages/ManageBookingPage/Calendar.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    //               onClick={() => props.changeDate(current)}
+    //             >
+    //               <span className={`text`}>{current.format('D')}</span>
+    //             </div>
+    //           );
+    //         })}
+    //     </div>
+    //   );
+    // }
+    // return calendar;
```
Comment (sina-jamshidi) [inline client/src/pages/ManageBookingPage/Calendar.js:76]: we don't need these

Comment (shawnohn) [reply client/src/pages/ManageBookingPage/Calendar.js:76]: yes, I will remove and commit all this kind of on further works

File: client/src/pages/ManageBookingPage/Calendar.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import './Calendar.scss';
+// import { MdChevronLeft, MdChevronRight } from 'react-icons/md';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import moment, { Moment as MomentTypes } from 'moment';
+
+function Calendar() {
```
Comment (sina-jamshidi) [inline client/src/pages/ManageBookingPage/Calendar.js:8]: cool! This is an interesting solution!

File: client/src/pages/ManageBookingPage/CurrentBookingComponent.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      />
+      <CardContent className={classes.contents}>
+        <Avatar
+          aria-label="recipe"
+          className={classes.avatar}
+          alt="T"
+          src="/images/profile_3.jpg"
+        />
+        <Typography variant="h6" className={classes.name} gutterBottom>
+          Jessica Pearson
```
Comment (sina-jamshidi) [inline client/src/pages/ManageBookingPage/CurrentBookingComponent.js:67]: Isn't that the boss character from Suits? lol

**Review by sina-jamshidi — COMMENTED** [review]
Great PR all around! Good clean code and good use of makeStyles. The screenshot looks great. Maybe tomorrow you can quickly show it running during standup

File: client/src/pages/ManageBookingPage/NextBookingComponent.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+export default function NextBookingComponent() {
+  const classes = useStyles();
+
+  return (
+    <Card className={classes.root}>
+      <CardHeader
+        action={
+          <IconButton aria-label="settings">
+            <SettingsIcon />
+          </IconButton>
```
Comment (sina-jamshidi) [inline client/src/pages/ManageBookingPage/NextBookingComponent.js:47]: So I know this button is in the mockups, but I'm not sure what we would actually use it for. Any ideas? If we can't think of anything I think you can remove it

Comment (shawnohn) [reply client/src/pages/ManageBookingPage/NextBookingComponent.js:47]: This button functions for modify booking which is I'm working on

File: client/src/pages/ManageBookingPage/Calendar.js (hunk 2 above)
Comment (shawnohn) [reply client/src/pages/ManageBookingPage/Calendar.js:8]: Thank you! but it's hard one to control though

**Review by TongilKim — APPROVED** [review]

**Pull request merged**